docs/tutorials/features/graph_optimization.md

```diff
@@ -13,13 +13,22 @@ ipex.enable_onednn_fusion(False)
 
 ### FP32 and BF16 models
 ```
+from dataclasses import dataclass
+
 import torch
 import torchvision.models as models
 
 # Import the Intel Extension for PyTorch
 import intel_extension_for_pytorch as ipex
 
-model = models.__dict__["resnet50 "](pretrained=True)
+
+@dataclass
+class Args:
+    batch_size: int
+
+args = Args(batch_size=4)
```

Comment:
Seems like `args` is not used. Why adding it?

Reply:
It is used later in the code.